### DISCUSSION TITLE
{"error":{"code":500,"status":"Internal Server Error","message":"sqlite create: named insert: unable to open database file: no such file or directory"}} when visiting the ui

### DISCUSSION THREAD

**Discussion by toxinliquid** [Security-Onion-Solutions/securityonion#11977] [2.4] [answered]
### Version

2.4.2

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

20

### Storage for /

250

### Storage for /nsm

50

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

{"error":{"code":500,"status":"Internal Server Error","message":"sqlite create: named insert: unable to open database file: no such file or directory"}} when visiting the ui

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by toxinliquid** [comment]
after reboot it works great , then after few hours this error appears again

**Comment by dougburks** [comment, marked as answer]
You specify that you're using version 2.4.2. Is that correct? That's an old Beta version from many months ago:
https://docs.securityonion.net/en/2.4/release-notes.html#beta-3-20230531-changes

Please download the latest version and perform a fresh installation:
https://securityonion.net/download
